Makefile

```diff
@@ -0,0 +1,13 @@
+create:
+	docker-compose build development
+
+clean:
+	docker system prune -f
+	docker-compose stop
+	docker rmi `docker images -a -q`
+
+fresh:
+	docker-compose build --no-cache development
+
+run:
+	docker-compose run --rm development
```

Comment:
It looks like this runs the container synchronously. Is there a way to run it in the background (i.e. the `-d` option)?